README.md

```diff
@@ -84,6 +84,13 @@ tools:
 Please see [`CONTRIBUTING.md`](CONTRIBUTING.md) for more information, including
 CLAs and best practices for working with GitHub.
 
+## Built with JanusGraph
```

Comment:
How about rephrase it to `Powered by JanusGraph` ?

Sounded more as stronger statement? ;)

Reply:
Updated; both seem fine to me.